workloads/kube-burner-ocp-wrapper/metrics-profiles/hosted-cluster-metrics.yml

```diff
@@ -16,6 +16,12 @@
 - query: (sum(container_memory_rss{name!="",container!="POD",namespace=~"openshift-(sdn|ovn-kubernetes|ingress|monitoring|image-registry)"}) by (container, pod, namespace, node) and on (node) kube_node_role{role="infra"}) > 0
   metricName: containerMemory-Infra
 
+- query: (avg(irate(container_cpu_usage_seconds_total{name!="",container!="POD",namespace=~"openshift-(etcd|.*apiserver|ovn-kubernetes|sdn|ingress|.*controller-manager|.*scheduler|image-registry)"}[2m]) * 100) by (namespace, pod, container)) > 0
+  metricName: containerCPU-AggregatedWorkers
+
+- query: avg(container_memory_rss{container!="POD",namespace=~"openshift-(etcd|.*apiserver|ovn-kubernetes|sdn|ingress|.*controller-manager|.*scheduler|image-registry)"}) by (pod, container, namespace)
```

Comment:
By having containerCPU-Workers and containerMemory-Workers I think we can skip these ones. You can add an extra query to the dashboard's pane to query for these metrics as well, so the old results won't break